includes/class-sensei-settings.php

```diff
@@ -861,31 +861,20 @@ public function render_learning_mode_setting( $args ) {
 		$value         = $options[ $key ];
 		$customize_url = Sensei_Course_Theme::get_sensei_theme_customize_url();
 		?>
-		<div class="sensei-settings-learning-mode__container">
-			<div class="sensei-settings-learning-mode__checkbox">
-				<input id="<?php echo esc_attr( $args['key'] ); ?>" name="<?php echo esc_attr( "{$this->token}[{$key}]" ); ?>" type="checkbox" value="1" <?php checked( $value, '1' ); ?> />
-				<label for="<?php echo esc_attr( $args['key'] ); ?>">
-					<?php esc_html_e( 'Enable for all courses', 'sensei-lms' ); ?>
-				</label>
-			</div>
-			<br />
-			<p class="sensei-settings-learning-mode__description">
-				<?php echo esc_html( $args['data']['description'] ); ?>
-			</p>
-			<?php if ( $customize_url ) { ?>
-			<br />
-			<p class="sensei-settings-learning-mode__description">
-				<?php esc_html_e( 'Customize the colors, add a logo, and other styling options for Sensei’s learning mode.', 'sensei-lms' ); ?>
-			</p>
-			<br />
-			<p class="sensei-settings-learning-mode__description">
-				<a class="sensei-settings-learning-mode__customize-link button" href="<?php echo esc_url( $customize_url ); ?>">
-					<?php esc_html_e( 'Customize styles', 'sensei-lms' ); ?>
-				</a>
-			</p>
-			<?php } ?>
-			<br />
-		</div>
+		<label for="<?php echo esc_attr( $args['key'] ); ?>">
```

Comment:
It contains a small refactor in the HTML to match the HTML of the other settings.